packages/rs-drive-abci/src/config.rs

```diff
@@ -8,13 +8,21 @@ pub struct PlatformConfig {
 
     /// Should we verify sum trees? Useful to set as no for tests
     pub verify_sum_trees: bool,
+
+    /// The default quorum size
+    pub quorum_size: u16,
```

Comment:
We have already `validator_set_quorum_hash`. Let's call this one `validator_set_quorum_size`?